hopsworks-common/src/main/java/io/hops/hopsworks/common/dao/jupyter/config/JupyterProcessMgr.java

```diff
@@ -242,12 +258,29 @@ public void stopJupyterServer(Project project, Users user, String hdfsUsername,
   public void projectCleanup(Project project) {
     projectCleanup(settings, LOGGER, osProcessExecutor, project);
   }
-
+  
   @Override
   @TransactionAttribute(TransactionAttributeType.NOT_SUPPORTED)
   public boolean pingServerJupyterUser(JupyterProject jupyterProject) {
```

Comment:
Can probably change this to some other name like "pingJupyterServer" or something, as we don't do it as "jupyter" user anymore.

Reply:
I changed it to `ping`. I will also make a PR to change the name in EE too